furbysource.asm

```diff
@@ -506,4 +506,285 @@ InitIO:
       STA   Mili_sec    ;also preset IRQ timer
 
         CLI             ;Enable IRQ
-A-20
\ No newline at end of file
+A-20
+      JSR   Kick_IRQ    ;wait for interrupt to restart
+      JSR   TI_reset    ;go init TI (uses 'Cycle_timer')
+
+; Preset motor speed, assuming mid battery life, we eat the pulse width
+; so that the motor wont be runing at 6 volts and burn out. We then
+; predict what the pulse width should be for any voltage.
+
+;     LDA   #Mpulse_on  ;preset motor speed
+      LDA   #ll
+      STA   Mon_len           ;set motor on pulse timing
+
+      LDA   #05         ;
+      STA   Moff_len    ;
```

Comment:
I think this line might need to be:

          STA   Moff_len    ;set motor off pulse timing